ops/ansible/playbooks-ccs/roles/platinum/tasks/install-dependencies.yml

```diff
@@ -28,3 +28,17 @@
       - tar
       - unzip
       - yum-utils
+      - yum-plugin-versionlock
```

Comment:
👋 Hey, @StewGoin... we're adding the versionlock plugin here. This seems to be innocuous, but thought I'd loop you and @sb-benohe in just the same.